src/components/select/Select.stories.tsx

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ import "styles/components/Select.scss";
+ import {storiesOf} from "@storybook/react";
+ import {action} from "@storybook/addon-actions";
+ import {Select, SelectOption, SelectOptionGroup} from "@dellstorage/clarity-react/forms/select";
+ 
+ storiesOf("Select", module)
+     .add("Basic select", () => (
+         <div>
+             <Select onChange={action("basic change")}>
+                 <SelectOption value="1">One</SelectOption>
+                 <SelectOption value="2">Two</SelectOption>
+                 <SelectOption value="3">Three</SelectOption>
+             </Select>
+         </div>
+     ))
+     .add("Basic select with select option", () => (
+         <div>
+             <Select onChange={action("basic change")}>
+                 <SelectOption value="select">select</SelectOption>
+                 <SelectOption value="1">One</SelectOption>
+                 <SelectOption value="2">Two</SelectOption>
+                 <SelectOption value="3">Three</SelectOption>
+             </Select>
+         </div>
+     ))
+     .add("Select with custom width", () => (
+         <div>
+             <Select onChange={action("basic change")} width="30%" label="Basic Select">
+                 <SelectOption value="1">One</SelectOption>
+                 <SelectOption value="2">Two</SelectOption>
+                 <SelectOption value="3">Three</SelectOption>
+             </Select>
+ 
+             <br />
+             {/* <Select onChange={action("basic change")} width="50%" isBoxed label="Boxed Select" className="boxedSelect">
+                 <SelectOption value="1">One</SelectOption>
+                 <SelectOption value="2">Two</SelectOption>
+                 <SelectOption value="3">Three</SelectOption>
+             </Select> */}
+         </div>
+     ))
+     .add("Labels", () => (
+         <div>
+             <Select onChange={action("label select - change")} label="I've got some options">
+                 <SelectOption value="1">One</SelectOption>
+                 <SelectOption value="2">Two</SelectOption>
+                 <SelectOption value="3">Three</SelectOption>
```

Comment:
This code is repeating. can we create some constant/function for this ?

Reply:
done